uber/templates/regform.html

```diff
@@ -181,8 +181,42 @@
                     <input type="hidden" name="amount_extra" value="{{ attendee.amount_extra }}" />
                 {% else %}
                     {% if c.DONATION_SLIDER_DISPLAY %}
-                        <div class="btn-group btn-group-vertical" data-toggle="buttons">
-                            {% radiogroup c.PREREG_DONATION_OPTS attendee.amount_extra %}
+
+                        <div data-toggle="buttons">
+                        {% for tier in c.PREREG_DONATION_DESCRIPTIONS %}
+                            <!-- TODO: use attendee.amount_extra to select correct or default tier -->
+                            <!-- TODO: deal with 'no thanks' $0 tier somehow -->
+                            <label class="btn btn-default" style="text-align: left;">
+                                <input type="radio"
+                                       name="amount_extra"
+                                       autocomplete="off"
+                                       value="{{ tier.price }}"
+                                       onchange="donationChanged();"
+                                       {% if attendee.amount_extra == tier.price %}checked{% endif %}  />
+                                <h4 style="text-align: center; text-transform: uppercase;">
+                                    {{ tier.name }}
+                                </h4>
+                                <span style="text-align: center; font-size: 120%; font-weight: bold;">
+                                <img src="{{ tier.icon }}" alt=""> + ${{ tier.price }}
+                            </span>
+                                <br>
+                                <button style="width: 100%;  margin-top: 1em;" class="btn btn-primary">
+                                    Select
+                                </button>
+                                <ul style="list-style-type: none; margin-top: 1em; padding-left: 0; font-size: 80%">
+                                    <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
+                                        Sweet Swag
```

Comment:
Correct, this is placeholder

On Aug 10, 2016 7:22 AM, "Victoria Earl" <notifications@github.com> wrote:

> In uber/templates/regform.html
> <https://github.com/magfest/ubersystem/pull/1946#discussion_r74226664>:
>
> > +                                       value="{{ tier.price }}"
> > +                                       onchange="donationChanged();"
> > +                                       {% if attendee.amount_extra == tier.price %}checked{% endif %}  />
> > +                                <h4 style="text-align: center; text-transform: uppercase;">
> > +                                    {{ tier.name }}
> > +                                </h4>
> > +                                <span style="text-align: center; font-size: 120%; font-weight: bold;">
> > +                                <img src="{{ tier.icon }}" alt=""> + ${{ tier.price }}
> > +                            </span>
> > +                                <br>
> > +                                <button style="width: 100%;  margin-top: 1em;" class="btn btn-primary">
> > +                                    Select
> > +                                </button>
> > +                                <ul style="list-style-type: none; margin-top: 1em; padding-left: 0; font-size: 80%">
> > +                                    <li style="padding: 0.5em; text-align: center; border-bottom: 1px solid whitesmoke;">
> > +                                        Sweet Swag
>
> Are these going to be replaced with a loop that lists items dynamically?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1946/files/e6294aa16a6238b7bc251e5ddabd357993a7fa2a#r74226664>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFKYyAa_pgPAFydzdjIGgtMuY_wa9HxDks5qebR7gaJpZM4JgwDl>
> .
>